from src.utils.Base.BaseException import CustomBaseError


class NotFoundError(CustomBaseError):
    pass
